src/main/kotlin/day1/Day1Suggestion.kt

```diff
@@ -0,0 +1,19 @@
+package day1
+
+fun main() {
+
+    val sumList = list.map {
+        it.sumOf { it.toInt() }
+    }
+    // region puzzle1
+    println(sumList.max())
+    // endregion
+
+    // region puzzle2
+    val result = sumList
+        .sortedDescending()
+        .subList(0, 3)
```

Comment:
saw this idea in the solution of Julia
```suggestion
        .take(3)
```